Formula/stanford-ner.rb

```diff
@@ -0,0 +1,20 @@
+class StanfordNer < Formula
+  desc "Stanford NLP Group's implementation of a Named Entity Recognizer."
+  homepage "http://nlp.stanford.edu/software/CRF-NER.shtml"
+  url "http://nlp.stanford.edu/software/stanford-ner-2015-04-20.zip"
+  version "3.5.2"
```

Comment:
I tried to visit the homepage but for some reason it doesn't load here so I just thought I'd ask presuming you already know :wink:. Just wanted to establish the versioning was going to be consistent, we struggle with changes in versioning systems.